### PR TITLE
Remove all mention of devpi, which we no longer use

### PR DESCRIPTION
**Pull request by nedbat**
Seems like we stopped using devpi with #971 two years ago.

### CODE REVIEW

**Comment by edx-webhook** [comment]
## PR Cover Letter
- [ ] A detailed description of the changes in the body of the PR
- [ ] If available, a link to the relevant JIRA ticket in the body of the PR
  - `[PROJ-XXXX](https://openedx.atlassian.net/browse/PROJ-XXXX)`

## Sandbox
- [ ] Build a sandbox for your branch and add a link here

## Testing
- [ ] i18n
- [ ] RTL
- [ ] Unit, integration, acceptance tests as appropriate
- [ ] Analytics
- [ ] Performance
- [ ] Database migrations are backwards-compatible

## Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [ ] Code review #1
- [ ] Code review #2
- [ ] Docs review (required for UI strings/error messages)
- [ ] UX review
- [ ] Accessibility review
- [ ] Product review

## Documentation considerations
- [ ] Are there user visible changes? If so, list who is affected, and include a pointer to the relevant code.
- [ ] Should this be mentioned in the release notes? If so, make sure that you included a description.

## DevOps assistance
If it's needed, or even if you suspect it might be needed, do any/all these things as early as possible. Don't surprise devops right before merging! For example, touching envs/aws.py or or envs/common.py are dead giveaways that you'll need assistance when pushing your changes out. Things that require devops attention include, but are not limited to:
- configuration changes
- feature flags
- migrations
Ways to get devops attention, pick-and-choose whichever is most appropriate for your specific PR:
- [ ] Tag devops on your PR (`@edx/devops`), either for review (more specific to devops = better) or just a general FYI
- [ ] File a support ticket

FYI: Tag anyone who might be interested in this PR here.

## Post-review
- [ ] Squash commits

File: playbooks/roles/stop_all_edx_services/handlers/main.yml [hunk 1]
```diff
@@ -36,9 +33,6 @@
 - name: stop memcached
   service: name=memcached state=stopped arguments="{{ STOP_ALL_EDX_SERVICES_EXTRA_ARGS }}"
 
-- name: stop supervisor.devpi
-  service: name=supervisor.devpi state=stopped arguments="{{ STOP_ALL_EDX_SERVICES_EXTRA_ARGS }}"
-
```
Comment (nedbat) [inline playbooks/roles/stop_all_edx_services/handlers/main.yml:41]: lol, good thing we stopped it twice. :)

**Comment by nedbat** [comment]
I was pretty brusque here, and may have missed some subtlety.  @edx/devops, I'm glad to be proved wrong.

**Comment by feanil** [comment]
@nedbat can you make sure that you can make sandboxes with this branch and that it doesn't cause any issues.  More detailed testing instructions here: https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist

**Comment by nedbat** [comment]
Sure thing.  First build: http://jenkins.edx.org:8080/job/ansible-provision/6374

**Comment by nedbat** [comment]
Second build (done with the exact parameters on the wiki page): http://jenkins.edx.org:8080/job/ansible-provision/6390

**Comment by nedbat** [comment]
Successful from-scratch build: http://jenkins.edx.org:8080/job/ansible-provision/6403/

**Comment by nedbat** [comment]
I also built the two AMIs with alton.

**Comment by nedbat** [comment]
@edx/devops any concerns?

**Comment by feanil** [comment]
:+1:

**Comment by jibsheet** [comment]
:+1:

**Comment by nedbat** [comment]
Rebased to fix conflicts (Max changed wiki links in many files, including ones I deleted).

**Pull request merged**